### DISCUSSION TITLE
Bundless 模式下, 如何指定某一个包为 Bundle 模式, 直接打进源码?

### DISCUSSION THREAD

**Discussion by lijinke666** [umijs/father#764] [Q&A]
版本: ("father": "^4.4.4")

在 monorepo 中我有如下子包, 其中 `s2-shared` 是一个公用包, 不会发布

![image](https://github.com/umijs/father/assets/21015895/f0459fb9-85bf-4af5-a02d-a8e5b7cc03bc)

比如 `s2-react` 依赖 `s2-shared`, 当 `s2-react` 以 bundless 的方式打包时, 我希望 `s2-shared` 能被直接打入源码, 这种能做到吗?

![image](https://github.com/umijs/father/assets/21015895/bff29f0e-a8c6-401d-8547-c40bcfbf1244)

**Comment by fz6m** [comment]
不可以，但是可以用 bundle 模式，然后把不需要打包的配成 external 试试。
